x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -392,6 +393,22 @@ const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
       properties: {},
     },
   },
+  [UNINSTALL_TOKENS_SAVED_OBJECT_TYPE]: {
+    name: UNINSTALL_TOKENS_SAVED_OBJECT_TYPE,
```

Comment:
Too late for the review, but this new SO type should be added to the new `.kibana_ingest` SO index instead.
Can be done by specifying `indexPattern: INGEST_SAVED_OBJECT_INDEX,`.

Reply:
👍 will open a PR to add.